File: CloudFlow/FrontEnd-App/src/app/shared/model/auth/login-response.model.ts
import { JsonObject, JsonProperty } from "json2typescript";
import { LoggedInUserModel } from "./logged-in-user.model";

@JsonObject
export class LoginResponseModel {

	@JsonProperty("authToken", String)
	public authToken: string = null;

	@JsonProperty("loggedInUser", LoggedInUserModel)
	public loggedInUser: LoggedInUserModel = null;
}

File: CloudFlow/src/ro/taxiApp/docs/presentation/client/shared/model/auth/LoginResponseModel.java
package ro.taxiApp.docs.presentation.client.shared.model.auth;

public class LoginResponseModel {

	private String authToken;
	private LoggedInUserModel loggedInUser;
	
	public String getAuthToken() {
		return authToken;
	}
	
	public void setAuthToken(String authToken) {
		this.authToken = authToken;
	}
	
	public LoggedInUserModel getLoggedInUser() {
		return loggedInUser;
	}
	
	public void setLoggedInUser(LoggedInUserModel loggedInUser) {
		this.loggedInUser = loggedInUser;
	}
}
